cnd/src/main.rs

```diff
@@ -52,7 +52,7 @@ fn main() -> anyhow::Result<()> {
     let seed = RootSeed::from_dir_or_generate(&settings.data.dir, OsRng)?;
 
     let mut runtime = runtime::Builder::new()
-        .stack_size(1024 * 1024 * 8) // the default is 2MB but that causes a segfault for some reason
+        .thread_stack_size(1024 * 1024 * 8) // the default is 2MB but that causes a segfault for some reason
```

Comment:
📣 Please clarify if you have tried reducing it.

Reply:
I didn't yet :)